test/zzz-rsyslog.sh

```diff
@@ -63,6 +63,9 @@ cleanup() {
         start_fluentd true 2>&1 | artifact_out
     else
         rpod=$( get_running_pod rsyslog )
+        oc logs $rpod -c rsyslog > $ARTIFACT_DIR/cleanup.rsyslog.log 2>&1 || :
+        oc exec $rpod -c rsyslog -- logs >> $ARTIFACT_DIR/cleanup.rsyslog.log 2>&1 || :
+        oc exec $rpod -c logrotate -- /usr/bin/cat /var/log/rsyslog/logrotate.log > $ARTIFACT_DIR/cleanup.logrotate.log 2>&1 || :
```

Comment:
you should use `get_rsyslog_pod_log` here

Reply:
Yeah, get_rsyslog_pod_log is called via get_all_logging_pod_logs at line 50.  I added them (line 66-68) to get them manually. :)  Once I could make zzz-rsyslog.sh work, I'm deleting these...
```
47 cleanup() {
 ...
 50     get_all_logging_pod_logs
```